packages/web-forms/src/components/FormHeader.vue

```diff
@@ -28,4 +97,32 @@ defineProps<{title: string}>();
 		}
 	}
 }
+
+.smaller-screens {
+	// this adds border to parent div so that bottom shadow of immediate child
+	// is visible.
+	border-bottom: 3px solid #fff;
+	
+	// to show ellipsis in the heading
+	// https://css-tricks.com/using-flexbox-and-text-ellipsis-together/
+	min-width: 0;
+
+	> div {
+		box-shadow: 0px 1px 2px 0px #0000004D;
+	}
+
+	h1 {
+		padding-left: 10px;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+		overflow: hidden;
+		font-size: 1.5rem;
+	}
+	
+	.btn-menu{
+		color: #000;
+		font-size: 1.5rem;
```

Comment:
- This color seems to affect only the hamburger icon, and doesn't seem to be dependent on screen size. Can we apply the color directly to that element (with a class identifying it as such)?

- The font size does not seem to affect anything, as far as I can tell. The same size is applied with higher precedence on `.p-button.p-button-icon-only.p-button-rounded`.

Reply:
good catch, I have removed it.